fill/actuators/qd.cc

```diff
@@ -1,19 +1,32 @@
 #include "qd.h"
 
-QD::QD(){
-
+QD::QD() {
+    stepperMotor = new StepperMotor(COIL1, COIL3);
 }
 
 QD::~QD(){
-
+    delete stepperMotor;
 }
 
 bool QD::Actuate(){
-    return true; 
+    if (actuated){
+        return false; // sensor did not actuate 
+    }
+
+    stepperMotor->Enable(); // enable sensor motor 
+    stepperMotor->Rotate(stepperMotor->CLOCKWISE, 50); // Rotate the motor clockwise, 50ms time delay
+
+    delay(60000); // We need 1200 steps
+
+    stepperMotor->StopRotation(); // Holds in current location 
+    stepperMotor->Disable();
+
+    actuated = true;
+    return actuated;
```

Comment:
This should just return true since we successfully start actuating. Technically we should separate that from what the actuated variable represents

Reply:
Because the QD only needs to be actuated one time, I thought that the behavior of the actuated variable would be to show that this one time process has occurred. What did you have in mind for the behavior of actuated?